src/polytope/models/ProblemSolution.py

```diff
@@ -0,0 +1,44 @@
+from dataclasses import dataclass
+from enum import Enum, auto
+
+from .SourceCode import SourceCode
+
+
+class ProblemSolutionType(Enum):
+    """! Types of problem solutions enumeration class."""
+
+    # Main model correct.
+    ModelCorrect = auto()
```

Comment:
(nit) `ModelCorrect` seems to be a bit awkward. Can we think of `MainCorrectSolution` or `ModelSolution`?

Reply:
I changed its name. Since the struct name is `ProblemSolutionType`, I did not add `Solution` to its name again.

https://github.com/Studio-Polytope/Polytope/blob/940bf14f32bf3c02c87d656bdfbbcbec2db826ba/src/polytope/models/ProblemSolution.py#L7-L16